api/valor_api/backend/core/llm_clients.py

```diff
@@ -26,6 +26,87 @@ class Messages(BaseModel):
     messages: list[Message]
 
 
+def _generate_claims_instruction(text: str) -> str:
+    """
+    Instruction template was copied from DeepEval's codebase https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/faithfulness/template.py.
+
+    Parameters
+    ----------
+    text: str
+        The text to extract claims from.
+
+    Returns
+    -------
+    str
+        The instruction for the llm.
+    """
+    return f"""Based on the given text, please generate a comprehensive list of FACTUAL claims that can inferred from the provided text.
+
+**
+IMPORTANT: Please make sure to only return in JSON format, with the "claims" key as a list of strings. No words or explanation is needed.
```

Comment:
can these instructions be added to one of the docstrings? it looks like they are sitting outside of a function at the moment.